pkg/apis/resource/validation/validation_resourceclaim_test.go

```diff
@@ -552,7 +552,7 @@ func TestValidateClaimStatusUpdate(t *testing.T) {
 			},
 		},
 		"invalid-reserved-for-too-large": {
-			wantFailures: field.ErrorList{field.TooLongMaxLength(field.NewPath("status", "reservedFor"), resource.ResourceClaimReservedForMaxSize+1, resource.ResourceClaimReservedForMaxSize)},
+			wantFailures: field.ErrorList{field.TooMany(field.NewPath("status", "reservedFor"), resource.ResourceClaimReservedForMaxSize+1, resource.ResourceClaimReservedForMaxSize)},
```

Comment:
/hold

> field.TooMany(

is TooMany on purpose ?

Reply:
there are other two  field.TooMany

Reply:
Yes, those were counting items, not bytes